graphdatascience/arrow_client/v2/job_client.py

```diff
@@ -26,14 +32,29 @@ def run_job(client: AuthenticatedArrowClient, endpoint: str, config: dict[str, A
         single = deserialize_single(res)
         return JobIdConfig(**single).job_id
 
-    @staticmethod
-    def wait_for_job(client: AuthenticatedArrowClient, job_id: str) -> None:
+    def wait_for_job(self, client: AuthenticatedArrowClient, job_id: str, show_progress: bool) -> None:
```

Comment:
there is a bit of an inconsistency in naming of the parameters.

`show_progress` on a global level such as AuraGraphDataScience.
`log_progress` on a algo call level